src/clj/harja/kyselyt/tyomaapaivakirja.sql

```diff
@@ -9,6 +9,82 @@ SELECT
    sopimustyyppi, 
    floor(random() * 3 + 0)::int AS tila 
 FROM urakka ORDER BY alkupvm DESC LIMIT 50;
+
+
+-- name: hae-paivakirjalistaus
+SELECT t.id as tyomaapaivakirja_id, t.urakka_id, u.nimi as "urakka-nimi",
+       d::DATE as paivamaara, -- Otetaan generoinnin päivämäärä
+       -- Ihan hihasta vedetty tilan määritelmä.
```

Comment:
Kun työmaapäiväkirja puuttuu, olisiko silti hyvä listata kyseisen urakan nimi listaukseen? Esim. 

```suggestion
       COALESCE(u.nimi, (SELECT nimi FROM urakka where id = :urakka-id)) as "urakka-nimi",
```

Reply:
Mä en oo yhtään varma, että tuo urakan nimi pysyy tuossa ui:ssa loppuun asti. Mutta, hyvä pointti.
Lisäsin urakan nimen, vaikka itse työmaapäiväkirjaa ei olisikaan.